mod/ckeditor/views/default/ckeditor/init.php

```diff
@@ -7,12 +7,14 @@
 
 ?>
 <script>
-// This global variable must be set before the editor script loading.
-CKEDITOR_BASEPATH = elgg.config.wwwroot + 'mod/ckeditor/vendors/ckeditor/';
+require(['elgg'], function (elgg) {
+	// This global variable must be set before the editor script loading.
```

Comment:
I was looking at this yesterday. I think we need an easier way to define global JS parameters. I think triggering a hook from js/initialize_elgg would do the trick.
Each plugin could then add their values to elgg.config server-side.
The reason I was looking at it, is I needed elgg.ACCESS_PUBLIC in the js script, which is not defined for some reason.